### DISCUSSION TITLE
Disney Channel Canada East

### DISCUSSION THREAD

**Discussion by TylerAustin7** [iptv-org/community#683] [Q&A]
**Channel Name:** Disney Channel Canada
**Channel ID:** DisneyChannelEast.ca
**Country:** Canada
**Language:** English
**Website:** disneychannel.ca
**Stream URL (optional):** xxx
**Notes (optional):** xxx

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/ca.m3u#L102-L103
